states/kentucky/elections.toml

```diff
@@ -22,3 +22,28 @@ in_person_by = 2020-06-23
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+online_by = 2020-10-05T16:00:00
+received_by = 2020-10-05T16:00:00
+in_person_by = 2020-10-05T16:00:00
+sources = ["https://elect.ky.gov/Resources/Documents/2020%20Election%20Calendar%20with%20Primary%20Postponement.pdf"]
+
+[20201103.absentee.application]
+received_by = 2020-10-27
+sources = ["https://elect.ky.gov/Resources/Documents/2020%20Election%20Calendar%20with%20Primary%20Postponement.pdf"]
+
+[20201103.absentee]
+received_by = 2020-11-03T18:00:00
+sources = ["https://elect.ky.gov/Resources/Documents/2020%20Election%20Calendar%20with%20Primary%20Postponement.pdf"]
+
+[20201103.poll.early]
+in_person_start = 2020-10-13
+in_person_by = 2020-11-02
+sources = ["https://vrsws.sos.ky.gov/abrweb/"]
+
+[20201103.poll]
+in_person_start = 2020-11-03T06:00:00
+in_person_by = 2020-11-03T18:00:00
+received_by = 2020-11-03T18:00:00
```

Comment:
What does received_by mean here?

Reply:
Its a mistake. Good catch. The PDF has received by block on 11/3 but that is for absentee.